### DISCUSSION TITLE
Project roadmap & task tracking

### DISCUSSION THREAD

**Discussion by aleksanderwalczuk** [SauveJeanLuc/coffee-tasks#58] [General]
We could make use of GitHub projects to track tasks. Right now I find that a bit confusing as more than one users tackle the same issue.

I also wonder if we could assign T-shirts or any other estimate to make sure that blockers are being resolved on time.

Lastly I'd like to talk about milestones, as a fresh maintainer I'm not sure about the project direction.

WDYT?

**Comment by SauveJeanLuc** [comment]
That's a good idea @aleksanderwalczuk .

First, for different people working on the same issue. I think we should prioritize a PR that's raised by someone who's been assigned to it. In case, a long time passes without the assignee raising a PR for it, we should give room to others as well to work on them.

For the second suggestion, I think I'd need more clarification on it, I don't get the idea well.

And for the last point, The very first achievement expected from the project is to have the app working as a full To-Do list app.
It was created, having hactoberfest in mind, that's why it's flexible enough for its direction to be set by maintainers, designers and collaborators, so as to have an app that fits interest of people with difrt backgrounds.

Besides being a fully functioning To-Do list app, it is expected to be AI-Powered, things like auto-completion while creating tasks, giving suggestions on sorting important tasks, and many other feasible suggestions that are yet to be given by the community.
